packages/editable-commands/src/listeners/pluginMessageUpdate.ts

```diff
@@ -0,0 +1,25 @@
+import { Events, Listener, PieceContext } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+export class PluginListener extends Listener<typeof Events.MessageUpdate> {
+	public constructor(context: PieceContext) {
+		super(context, { event: Events.MessageUpdate });
+	}
+
+	public run(old: Message, message: Message) {
+		// If the contents of both messages are the same, return:
+		if (old.content === message.content) return;
+
+		// If the message was sent by a webhook, return:
+		if (message.webhookId !== null) return;
+
+		// If the message was sent by the Discord system, return:
+		if (message.system) return;
+
+		// If the message was sent by a bot, return:
+		if (message.author.bot) return;
+
+		// Run the message parser.
+		this.container.client.emit(Events.PreMessageParsed, message);
```

Comment:
Seeing as in Skyra we also emit yet another event at this stage I think it would be a good idea to also emit `pluginMessageUpdateAccepted` here that we can then listen to in Skyra. Otherwise we have to still duplicate this event in Skyra.

That also means making an exported `const enum` called `PluginEditableCommandsEvents`

Reply:
We'll just refactor Skyra's to use `preMessageParsed` instead of running the same logic twice.

Reply:
fair